src/transformers/integrations.py

```diff
@@ -1288,7 +1299,10 @@ def _log_model_checkpoint(self, source_directory: str, checkpoint: str):
         if self._volatile_checkpoints_dir is not None:
             consistent_checkpoint_path = os.path.join(self._volatile_checkpoints_dir, checkpoint)
             try:
-                shutil.copytree(relative_path, os.path.join(consistent_checkpoint_path, relative_path))
+                # Remove leading ../ from a relative path.
+                cpkt_path = relative_path.replace("..", "").lstrip("/")
```

Comment:
Like I said before, this needs to be OS-agnostic so you should use `os.path.sep` instead of `"/"`.

Reply:
Ah, I understand now! Thanks